sessions/motivation.qmd

```diff
@@ -0,0 +1,140 @@
+# Motivation
```

Comment:
Each "chapter" (or page on the website) must start with and only contain one `#` top level header. All other ones are `##` or more.